cmd/erasure-object.go

```diff
@@ -128,7 +128,7 @@ func (er erasureObjects) CopyObject(ctx context.Context, srcBucket, srcObject, d
 		return oi, toObjectErr(err, srcBucket, srcObject)
 	}
 
-	return fi.ToObjectInfo(srcBucket, srcObject), nil
```

Comment:
We shouldn't re-read the disk @poornas - whatever needs to be changed that is persisted should be present in the update `fi`